### PR TITLE
List of Changes

### PR DESCRIPTION
**Pull request by nmnarora600**
Added Full name of android R in android-versions.md

Added nested If-Else in c.md

Added JS 2019 details in javascript-versions.md

Corrected Conditional Statements section in python.md

Added some information in cpp.md

### CODE REVIEW

File: python.md [hunk 1]
```diff
@@ -176,7 +176,33 @@ elif conditional-expression :
 else :
     #code
 ```
-### 4. Try Except
+### 4. Nested-If-else
```
Comment (onecompiler-ops) [inline python.md:179]: Please remove this section.

File: python.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ if conditional-expression :
 else :
     #code
 ```
-### 3. Nested-If-else
+### 3. If-elif-else Ladder
```
Comment (onecompiler-ops) [inline python.md:169]: Looks good

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: android-versions.md [hunk 3]
```diff
@@ -25,6 +25,6 @@ color: #f4de50
 |8.0|August 21, 2017|Oreo|Notification improvements, Multi-display support, Wi-Fi Assistant |
 |9.0|August 6, 2018|Pie|Rounded corners across the UI, Android Dashboard, Redesigned volume slider |
 |10|September 3, 2019|Q|New safeguards to protect user privacy that need to support in our app. |
-|11|September 8, 2020|R|Conversations, bubbles, device and media controls, 5G, biometric and more.|
+|11|September 8, 2020|Red Velvet Cake|Conversations, bubbles, device and media controls, 5G, biometric and more.|
```
Comment (onecompiler-ops) [inline android-versions.md:28]: Please raise PR with one cheat sheet changes at a time

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: android-versions.md (hunk 3 above)
Comment (nmnarora600) [reply android-versions.md:28]: Since I am a newbie, I don't know how to do that.
All pr gets automatically merged to one.

**Comment by onecompiler-ops** [comment]
Create a new fork for each cheat sheet

**Comment by nmnarora600** [comment]
> Create a new fork for each cheat sheet

It says No more forks can be created. These forks already exist

**Pull request closed without merging**